bundles/org.openhab.binding.jablotron/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:jablotron:device">
+		<parameter name="refresh" type="integer" required="false" min="30" unit="s">
```

Comment:
Can you apply the min value and the unit to the other refresh parameters, too? Is 30 sec. a reasonable min value? Maybe the API gets faster some day?

Reply:
lowered to 10s and added to the missing